README.md

```diff
@@ -32,13 +32,15 @@ You will need to:
     - `Issues`
     - `Pull requests`
     - `Metadata` (should be set automatically when you set it for `Issues` or `Pull requests`)
+    - `Discussions` - currently not available with fine-grained PATs
 
 #### Generating a Classic PAT
 You will need to:
 - [open the Classic PAT generation page](https://github.com/settings/tokens/new)
 - select the following scopes:
     - `repo`: used for getting the repository information, such as last update date time on issues and pull requests
     - `read:user`: used for retrieving the current logged in user information _(GitHub ID and username)_
+    - `write:discussions`: used to have access to the discussions API
```

Comment:
why write?

Reply:
Yepp, sorry about that. I had both checkboxes checked when I created the classic PAT. It does indeed work with read:discussions and there is no need for write.